### PR TITLE
Update lava M chain.json

### PR DESCRIPTION
**Pull request by AviaOne**

### CODE REVIEW

File: lava/chain.json [hunk 1]
```diff
@@ -111,6 +116,10 @@
         "address": "https://lava-rpc.ibs.team:443",
         "provider": "Inter Blockchain Services"
       }
```
Comment (reversesigh) [inline lava/chain.json:118]: Add a comma here

File: lava/chain.json [hunk 2]
```diff
@@ -125,6 +134,10 @@
         "address": "https://lava-api.ibs.team:443",
         "provider": "Inter Blockchain Services"
       }
```
Comment (reversesigh) [inline lava/chain.json:136]: Add a comma here

**Review by AviaOne — COMMENTED** [review]
add comma

**Comment by AviaOne** [comment]
sorry for this mess !

**Pull request merged**